### ISSUE TITLE
Overriding the base url

### ISSUE THREAD

**Issue by patatepartie** [ianheggie/health_check#17]
From what I understand of the documentation and the code, it is not possible to override the base path used to access the checks.

Not using  `health_check_routes` and defining manually the route seems to work:
`get '/ping(/:checks)(.:format)', :to => 'health_check/health_check#index'`

Am I missing something ?
Otherwise it would be great to have a config option to override the base path, or at least to update the documentation explaining how to do it manually.

**Comment by edwinlunando** [comment]
I suffer from this too. I followed the documentation but it did not work.

**Comment by squarelover** [comment]
I'm having the same problem. ELB in AWS defaults to `/health`. The example in the README don't work.

**Comment by squarelover** [comment]
I figured out that this feature is in master, but not the released version. If I set my Gemfile to use the github repo, and the current master ref it works for me.

```
gem "health_check", :github => "ianheggie/health_check", :ref => "7839a6"
```

**Comment by ianheggie** [comment]
@squarelover: Released in v2.0.0 - use health_check_routes('health')
